utils/utils.go

```diff
@@ -160,3 +160,16 @@ func PopulateMetadataForBulkPublishEntry(reqMeta, entryMeta map[string]string) m
 
 	return resMeta
 }
+
+func Filter[T any](items []T, fn func(item T) bool, isFilterNotApplicable bool) []T {
+	if isFilterNotApplicable {
```

Comment:
Sure, that actually makes sense. What I had started to think was someone using `Filter` and forgetting about it, with some specific flags - but then instead of setting/re-setting the flag, they can chose to call/not-call Filter itself.

Reply:
Removed the flag.